### PR TITLE
Desktop version 

### PR DESCRIPTION
**Pull request by DiamondNath07**
### This is the desktop version of my portfolio

It was built with the following;
- Index.html
- Styles.css
- linters.yml
 This version host a different background from the mobile version.

It is however responsive friendly.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -218,27 +296,28 @@ <h6>skills</h6>
...
+          <input email="email" type="email" required value="Email address" />
+        </label>
+        <label>
+          <textarea name="message" maxlength="500" required placeholder="Enter text here"></textarea>
+        </label>
+        <span>
+          <button type="submit">get in touch</button>
+        </span>
+      </form>
+    </div>
```
Comment (mckent05) [inline index.html:320]: - There are few differences between your contact and the design in the Figma template. I'd kindly suggest you try to fix this 🙏 . Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your design:
-- Please pay attention to the margin on the left side of the text, kindly fix this 👍 
-- Your contact form doesn't exactly match that of the design (there are two input fields for the name) 👍 
-- Your button is aligned as it is in the Figma template, kindly fix this 😄 😄 
contact 5

Figma design:
contact figma2

- Also please fix your button alignment for the mobile version in your contact form 👍 
current contact mobile

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @DiamondNath07, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your project is gradually getting complete and it's looking quite nice 😄 😄 
- You have included a descriptive PR summary and PR title 👍 👍 

## Required Changes ♻️
## There are some issues I found with some parts of your mobile version, Please kindly fix the issue
- The texts in the about myself section in your portfolio are overlapping. Kindly fix this issue 😄 

Your project:
current about me

- The buttons in your project cards are not properly aligned, kindly please this issue for all your project cards 🙏 🙏 

Your project:
figma 2 project card mobile


_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: utils.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        <div class="text-wrapper ">
+          <h4>Professional Art Printing Data</h4>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+            industry's standard</p>
+          <button id="btn" type="button">html</button>
+          <button id="btn" type="button">bootstrap</button>
+          <button id="btn" type="button">Ruby</button>
+          <div class="card-button">
+            <a href="#"><button type="submit">See Project</button></a>
+          </div>
```
Comment (bobb-Rob) [inline utils.html]: - [ ] [REQUIRED] The buttons are not showing on your desktop view on hover. Please note that the button should appear upon mouse hover. In other words, the projects cards before mouse hover have no button, but upon hover the button appears. Kindly implement that.

**Initial state** - When the project card is not hovered on with the mouse


**On hover** - When the project is hovered on with the mouse



image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @DiamondNath07 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -18,151 +16,250 @@
...
+        <ul class="lang">
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">ruby</a></li>
+        </ul>
+        </div>
+        <div class="caad-btn">
+          <button type="button">See Project</button>
+        </div>
+          </div>
```
Comment (awais-amjed) [inline index.html:117]: As mentioned by the previous reviewer, kindly make sure that the button should appear only upon mouse hover. I am attaching an image below explaining how it should work 🤗

image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @DiamondNath07 , 

Great work on making the changes requested by a previous reviewer 👏👏👏
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (DiamondNath07) [reply index.html:117]: Please kindky understand that the hover effect required of me here is not possible with css as most of the prople who have done this project got a pass without this. I'm almost close to the deadline pls consider.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @DiamondNath07 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost2

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 
- I appreciate your effort, you can do it! ✔️ 

 ## Required Changes ♻️

image

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — COMMENTED** [review]
Hello @DiamondNath07 👋,

_This is an additional review due to the second opinion request._

The code reviewer's only concern is that your app in its current state is unusable. Even if you do not use JavaScript for the hover effect, you still need to ensure that all the cards have the "See Project" button and are styled as in the provided design.

As the code reviewer has pointed out, one card's button is not visible and the other's button is not correctly styled. Please refactor the UI's design and ask for another code review.

Keep rocking 🚀!

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @DiamondNath07, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far on addressing the issue! 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**